functional-tests/src/main/scala/com/gu/membership/pages/EventsListPage.scala

```diff
@@ -31,6 +23,6 @@ class EventsListPage(driver: WebDriver) extends BaseMembershipPage(driver) {
   }
 
   def clickAnEvent(): EventPage = {
-    clickEventByIndex(10)
+    clickEventByIndex(3)
```

Comment:
As `eventLinkList` is now only a list of bookable events (active and not sold out) this should avoid you needing to manually change this number. Do we still need to click by index in this case, or could we perhaps select a random available event?